libqtile/scripts/start.py

```diff
@@ -95,14 +103,23 @@ def start(options):
 
 def add_subcommand(subparsers, parents):
     parser = subparsers.add_parser("start", parents=parents, help="Start a Qtile session.")
-    parser.add_argument(
+    configs = parser.add_mutually_exclusive_group()
+    configs.add_argument(
         "-c",
         "--config",
         action="store",
         default=get_config_file(),
         dest="configfile",
         help="Use the specified configuration file.",
     )
+    configs.add_argument(
+        "-d",
+        "--default",
```

Comment:
Does argparse render these config groups somehow? otherwise maybe `--use-default-config` instead, so it's clear what is defaulting to what?